### DISCUSSION TITLE
Does pulumi support max-pods-per-node?

### DISCUSSION THREAD

**Discussion by raghulkrishna** [pulumi/pulumi#7519] [Pulumi Providers] [answered]

**Comment by lukehoban** [comment]
@raghulkrishna Which resource are you trying to use?

**Reply by raghulkrishna** [reply]
aws eks

**Comment by lukehoban** [comment]
Do you have a link to the AWS documentation for the feature you are looking to use?

**Comment by nimbinatus** [comment, marked as answer]
This question is similar to https://github.com/pulumi/pulumi-eks/issues/609, which has more context provided like links to the documentation. Let's use that issue to continue this discussion and keep it all in one place. Thanks!
